Missing tester dependency when running docker

* Version: 4.3.0
* Python: 3.6
* OS: linux


### What was wrong?

Following instructions on README.MD, I tried to run tests using docker in sandbox. Got the following errors:

```
______________________________ ERROR collecting  _______________________________
/usr/local/lib/python3.6/site-packages/py/_path/common.py:377: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
/usr/local/lib/python3.6/site-packages/py/_path/common.py:429: in gen
    for p in self.gen(subdir):
/usr/local/lib/python3.6/site-packages/py/_path/common.py:418: in gen
    dirs = self.optsort([p for p in entries
/usr/local/lib/python3.6/site-packages/py/_path/common.py:419: in <listcomp>
    if p.check(dir=1) and (rec is None or rec(p))])
/usr/local/lib/python3.6/site-packages/_pytest/main.py:737: in _recurse
    ihook = self.gethookproxy(path)
/usr/local/lib/python3.6/site-packages/_pytest/main.py:641: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
/usr/local/lib/python3.6/site-packages/_pytest/config.py:356: in _getconftestmodules
    mod = self._importconftest(conftestpath)
/usr/local/lib/python3.6/site-packages/_pytest/config.py:381: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   _pytest.config.ConftestImportFailure: ModuleNotFoundError("No module named 'eth_tester'",)
E     File "/usr/local/lib/python3.6/site-packages/_pytest/assertion/rewrite.py", line 212, in load_module
E       py.builtin.exec_(co, mod.__dict__)
E     File "/code/tests/ens/conftest.py", line 5, in <module>
E       from eth_tester import (
=========================== 4 error in 4.04 seconds ============================
```

### How can it be fixed?

Fixed by manually run `docker-compose exec sandbox bash -c 'pip install -e .[tester]'`. Should this been reflected on README.MD as an additional step to run tests in docker?

--- a/Dockerfile
+++ b/Dockerfile
@@ -19,6 +19,6 @@ COPY ens ./ens/
 COPY setup.py .
 COPY README.md .
 
-RUN pip install -e .
+RUN pip install -e .[tester]
 
 WORKDIR /code

